osf/models/notification_type.py

```diff
@@ -22,7 +22,7 @@ class Type(str, Enum):
         DESK_ARCHIVE_REGISTRATION_STUCK = 'desk_archive_registration_stuck'
         DESK_REQUEST_EXPORT = 'desk_request_export'
         DESK_REQUEST_DEACTIVATION = 'desk_request_deactivation'
-        DESK_OSF_SUPPORT_EMAIL = 'desk_osf_support_email'
+        DESK_OSF_SUPPORT_EMAIL = 'desk_osf_support_email'  # unused same as DESK_CROSSREF_ERROR
```

Comment:
As discussed, we need another ticket to review all unused types. We found bugs where we used the wrong types which led to the correct ones unused.